source/convergence_test.cc

```diff
@@ -594,52 +536,88 @@ void ConvergenceTestData::format_columns()
   {
     table.set_scientific("L2", true);
     table.set_precision("L2", precision);
-    table.add_column_to_supercolumn("L2", "error norms");
   }
 
   if (H1_error_specified)
   {
     table.set_scientific("H1", true);
     table.set_precision("H1", precision);
-    table.add_column_to_supercolumn("L2", "error norms");
   }
 
   if (Linfty_error_specified)
   {
     table.set_scientific("Linfty", true);
     table.set_precision("Linfty", precision);
-    table.add_column_to_supercolumn("Linfty", "error norms");
   }
 
+
+  std::string column;
+  bool evaluate_convergence{false};
+
+  switch (type)
+  {
+    case ConvergenceTestType::spatial:
+      column.assign("h_max");
+      evaluate_convergence = true;
+      Assert(dimension > 0, ExcLowerRange(dimension, 0));
+      break;
+    case ConvergenceTestType::temporal:
+      column.assign("step size");
+      dimension = 1;
+      evaluate_convergence = true;
+      break;
+    case ConvergenceTestType::spatio_temporal:
+      break;
```

Comment:
Assert undefined `spatio_temporal` case.

Reply:
Added a comment in d423619.